aequilibrae/project/project.py

```diff
@@ -28,58 +32,97 @@ def __init__(self):
+
+        if os.path.isdir(project_path):
+            raise FileNotFoundError("Location already exists. Choose a different name or remove the existing directory")
         self.__create_empty_project()
+        self.__load_objects()
+        logger.info(f'Created project on {self.project_base_path}')
+
+    def close(self) -> None:
+        """Safely closes the project"""
+        if self.environ_var in os.environ:
+            self.conn.close()
+            os.environ.pop(self.environ_var, None)
+            logger.info(f'Closed project on {self.project_base_path}')
+        else:
+            warnings.warn('There is no Aequilibrae project open that you may close')
+
+    def load(self, project_path: str) -> None:
+        """
+        Loads project from disk
+
+        Args:
+            *project_path* (:obj:`str`): Full path to the project data folder. If the project inside does
+            not exist, it will fail.
+        """
+        warnings.warn(f"Function has been deprecated. Use my_project.open({project_path}) instead", DeprecationWarning)
+        self.open(project_path)
+
+    def __load_objects(self):
+        self.parameters = Parameters().parameters
 
-        self.conn = spatialite_connection(self.conn)
         self.network = Network(self)
+        os.environ[self.environ_var] = self.project_base_path
 
     def __create_empty_project(self):
+
+        # We create the project folder and create the base file
+        os.mkdir(self.project_base_path)
         shutil.copyfile(spatialite_database, self.path_to_file)
-        self.conn = sqlite3.connect(self.path_to_file)
-        self.__create_modes_table()
+        self.conn = spatialite_connection(sqlite3.connect(self.path_to_file))
 
-    def __create_modes_table(self):
+        # We create the enviroment variable with the the location for the project
+        os.environ[self.environ_var] = self.project_base_path
 
-        create_query = """CREATE TABLE 'modes' (mode_name VARCHAR UNIQUE NOT NULL,
-                                                mode_id VARCHAR PRIMARY KEY UNIQUE NOT NULL,
-                                                description VARCHAR);"""
+        # Write parameters to the project folder
+        p = Parameters()
+        p.parameters["system"]["logging_directory"] = self.project_base_path
+        p.write_back()
+
+        # Create actual tables
         cursor = self.conn.cursor()
-        cursor.execute(create_query)
-        modes = self.parameters["network"]["modes"]
-
-        for mode in modes:
-            nm = list(mode.keys())[0]
-            descr = mode[nm]["description"]
-            mode_id = mode[nm]["letter"]
-            par = [f'"{p}"' for p in [nm, mode_id, descr]]
-            par = ",".join(par)
-            sql = f"INSERT INTO 'modes' (mode_name, mode_id, description) VALUES({par})"
-            cursor.execute(sql)
+        cursor.execute('PRAGMA foreign_keys = ON;')
```

Comment:
did I show you the way we automatically setup connections to have this turned on?
```python
    engine = create_engine("sqlite:///{}".format(db_path))
    listen(engine, "connect", engine_listener)
```

where 

```python
def engine_listener(dbapi_conn, _connection_record):
    """Set up a listener to augment sqlalchemy connections with spatialite and foreign_key support."""
    add_mod_spatialite_path()
    dbapi_conn.enable_load_extension(True)
    dbapi_conn.load_extension("mod_spatialite")

    # Turn on foreign key checking
    cursor = dbapi_conn.cursor()

    cursor.execute("PRAGMA foreign_keys=ON")
    cursor.close()
```

Reply:
SQLAlchemy is not available with QGIS for Windows...